brainstormforce/astra
# AST-3321 - In all customizer color picker options circles showing in different format & Build: Release PR (WP6.3)

### Description
- Final release PR. Please check all the checklist below before merging.
---

### Checklist:
- [x] Finalize changelog.txt
- [x] Add doc links in the changelog.txt
- [x] Change minimum addon update version - ASTRA_EXT_MIN_VER
- [x] Run to update packages        : `npm audit fix`
- [x] Run to produce minified files : `grunt minify`
- [x] Run to update Google Fonts    : `grunt google-fonts`
- [x] PHPCS - vendor/bin/phpcs
- [x] Run to update version number  : `grunt version-bump --ver=<version-number>`
- [x] Check the Background Process function version
- [x] Update - Tested Upto - style.css & readme.txt
- [x] Update Readme.txt file and then run: `grunt readme`
- [x] Update - `theme-auto-version` : In `inc/compatibility/starter-content/astra-settings-export.json`
- [x] Generate POT file             : `wp i18n make-pot wp-content/themes/astra wp-content/themes/astra/languages/astra.pot`
- [x] Add draft Release notes on github
- [x] Add draft changelogs to wpastra.com
- [ ] Open PR to backport branch to `next-release`
---

### Description
- Managed UI with old versions for color picker control

### Screenshots
- https://d.pr/i/BuNdIO

### Types of changes
- Bug fix

### How has this been tested?
- Update setup to WP6.3
- Check any color control from the customizer

### Checklist:
